### PR TITLE
db/kpi-design

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Comment by bre1470** [comment]
There are two issues left afaik:
- The styledMode CSS prevents colored gauge because of `highcharts-color-0`
- I do not get the yAxis labels to be activated. My reference was this demo: https://www.highcharts.com/demo/gauge-solid

File: samples/dashboards/demos/climate/demo.css [hunk 1]
```diff
@@ -209,31 +201,79 @@
...
+    stroke: var(--color-pane-line);
+}
+
+#kpi-layout .hd-component-kpi-chart-container .highcharts-series .highcharts-color-0 {
+    fill: var(--color-gauge);
+    stroke: none;
+}
+
+#kpi-layout .hd-component-kpi-chart-container .highcharts-title {
+    font-size: 13px;
```
Comment (karolkolodziej) [inline samples/dashboards/demos/climate/demo.css:244]: I feel like the `font-size` might be slightly bigger. I can't see that in figma.

**Review by karolkolodziej — APPROVED** [review]

File: samples/dashboards/demos/climate/demo.css (hunk 1 above)
Comment (bre1470) [reply samples/dashboards/demos/climate/demo.css:244]: Probably, but we are out of space anyway.

**Review by sebastianbochan — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]
Indeed, something strange with these labels. We go with what we have, thanks!

Please merge in 👍

**Pull request merged**